canonical-web-and-design/ubuntu.com
# Update /core page

## Done

- Update `/core` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/core
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [copy doc](https://docs.google.com/document/d/1QoM0Yqx4JfDJ2G3OPt5bds-ximG1jASmU_PWX4iff3E/edit)


## Issue / Card

Fixes #6033 


P.S. Ignore the different image in the hero as the whole hero will be re-designed.
